src/services/transpile.ts

```diff
@@ -43,6 +43,12 @@ export interface TranspileOutput {
     sourceMapText?: string;
 }
 
+const optionsRedundantWithVerbatimModuleSyntax = new Set([
+    "isolatedModules",
+    "preserveValueImports",
+    "importsNotUsedAsValues"

```

Comment:
I don't think either of these can actually show up in `transpileOptionValueCompilerOptions` because neither explicitly set `transpileOptionValue`. But, I guess it's harmless if we want this PR in ASAP.

Reply:
I added them just to be on the safe side, since it would be fairly easy to miss if we made any changes in compilerOptions.